### DISCUSSION TITLE
Where is the latest  nedit-ng version ?

### DISCUSSION THREAD

**Discussion by ericxuo** [eteran/nedit-ng#316] [General]
Hi,

If I want to download the latest nedit-ng version, where should I take it from ?
1) https://github.com/eteran/nedit-ng/releases/tag/2020.1
2) https://sourceforge.net/p/nedit/git/ci/NEDIT-NG-START/tree/ (and how to download it from there ?)

I thought nedit and nedit-ng were about to merge to have one single development branch.

Regards.

Xuo.

**Comment by eteran** [comment]
The future of NEdit 6.0 is indeed NEdit -ng. The main hold up is that there are a select few outstanding quirks that I want to get resolved before calling it "NEdit 6.0", and unfortunately, this year has just been **VERY** busy in both work and personal life so I haven't had the cycles to put a bow on those last bits. I definitely want to get things across the finish line and am actively developing still... it's just very slow since I find "an hour here" and "an hour there" to do it.

So here's the complete story:

If you want the **latest** of NG, I would recommend that you just pull from the `master` branch here on Github. There has been a lot of fixes since the 2020.1 release and you probably want those included.

On Sourceforge, that branch is effectively a periodic mirror of the NG code here on GitHub with branding changed to be NEdit 6.0. So it will always be at best (more or less) exactly what you'd get from the Github source tree, but more likely, it's going to be on average several commits behind the master branch here.

Hope that clears things up for you.

**Comment by ericxuo** [comment]
Hi,

It's clear now. Thank you for the explanations.

Regards.

Xuo.
